GuillotineMenu/GuillotineMenuTransitionAnimation.swift

```diff
@@ -117,6 +123,7 @@ class GuillotineTransitionAnimation: NSObject {
         animationContext = context
         animator = UIDynamicAnimator(referenceView: context.containerView()!)
         animator.delegate = self
+        vectorDY = CGFloat(2/3.14 * (Double(UIScreen.mainScreen().bounds.size.height) / duration))
```

Comment:
Could you please make this expression as a private let if it doesn't change with time?